From fd4861b413c3e2e6a139a349293bf81425e33f0e Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 19 Sep 2014 12:29:22 -0400
Subject: [acpi] rename acpi_evaluate_hotplug_ost() to acpi_evaluate_ost()

Message-id: <1411129776-21656-48-git-send-email-prarit@redhat.com>
Patchwork-id: 96363
O-Subject: [RHEL7.1 PATCH BZ 1141846 47/61] ACPI: rename acpi_evaluate_hotplug_ost() to acpi_evaluate_ost()
Bugzilla: 1141846
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Myron Stowe <mstowe@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1141846

commit 05730c1952787faa422ea2898634a1f9cdb61161
Author: Jiang Liu <jiang.liu@linux.intel.com>
Date:   Wed Feb 19 14:02:15 2014 +0800

    ACPI: rename acpi_evaluate_hotplug_ost() to acpi_evaluate_ost()

    Rename acpi_evaluate_hotplug_ost() to acpi_evaluate_ost() for later
    resue.

    Signed-off-by: Jiang Liu <jiang.liu@linux.intel.com>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Lenny Szubowicz <lszubowi@redhat.com>
Cc: Tony Camuso <tcamuso@redhat.com>
Cc: Stefan Assmann <sassmann@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/acpi/utils.c b/drivers/acpi/utils.c
index 552248b..e2edf0d 100644
--- a/drivers/acpi/utils.c
+++ b/drivers/acpi/utils.c
@@ -430,17 +430,15 @@ EXPORT_SYMBOL(acpi_get_physical_device_location);
  * When the platform does not support _OST, this function has no effect.
  */
 acpi_status
-acpi_evaluate_hotplug_ost(acpi_handle handle, u32 source_event,
-  u32 status_code, struct acpi_buffer *status_buf)
+acpi_evaluate_ost(acpi_handle handle, u32 source_event, u32 status_code,
+    struct acpi_buffer *status_buf)
 {
-#ifdef ACPI_HOTPLUG_OST
  union acpi_object params[3] = {
   {.type = ACPI_TYPE_INTEGER,},
   {.type = ACPI_TYPE_INTEGER,},
   {.type = ACPI_TYPE_BUFFER,}
  };
  struct acpi_object_list arg_list = {3, params};
- acpi_status status;
 
  params[0].integer.value = source_event;
  params[1].integer.value = status_code;
@@ -452,13 +450,9 @@ acpi_evaluate_hotplug_ost(acpi_handle handle, u32 source_event,
   params[2].buffer.length = 0;
  }
 
- status = acpi_evaluate_object(handle, "_OST", &arg_list, NULL);
- return status;
-#else
- return AE_OK;
-#endif
+ return acpi_evaluate_object(handle, "_OST", &arg_list, NULL);
 }
-EXPORT_SYMBOL(acpi_evaluate_hotplug_ost);
+EXPORT_SYMBOL(acpi_evaluate_ost);
 
 /**
  * acpi_handle_printk: Print message with ACPI prefix and object path
diff --git a/include/acpi/acpi_bus.h b/include/acpi/acpi_bus.h
index 02308c2..dcdcd80 100644
--- a/include/acpi/acpi_bus.h
+++ b/include/acpi/acpi_bus.h
@@ -51,8 +51,16 @@ acpi_evaluate_reference(acpi_handle handle,
    struct acpi_object_list *arguments,
    struct acpi_handle_list *list);
 acpi_status
+acpi_evaluate_ost(acpi_handle handle, u32 source_event, u32 status_code,
+    struct acpi_buffer *status_buf);
+#ifdef ACPI_HOTPLUG_OST
+#define acpi_evaluate_hotplug_ost acpi_evaluate_ost
+#else
+static inline acpi_status
 acpi_evaluate_hotplug_ost(acpi_handle handle, u32 source_event,
-   u32 status_code, struct acpi_buffer *status_buf);
+   u32 status_code, struct acpi_buffer *status_buf)
+{ return AE_OK; }
+#endif
 
 acpi_status
 acpi_get_physical_device_location(acpi_handle handle, struct acpi_pld_info **pld);
-- 
1.7.1